ui/webui/test/helpers/installer.py

```diff
@@ -35,6 +35,9 @@ def next(self):
 
     def open(self):
         self.browser.open("/cockpit/@localhost/anaconda-webui/index.html#/installation-language")
+        # wait until the page is sufficiently initialized
```

Comment:
Tagging @velezd as I remember him mentioning some race-condition like behaviour when running some of the Web UI tests. Good to see it possibly fixed. :)